lib/src/config.dart

```diff
@@ -29,7 +29,8 @@ class Config {
           const TextStyle(fontSize: 20, color: Colors.black26),
       this.tabIndicatorAnimDuration = kTabScrollDuration,
       this.categoryIcons = const CategoryIcons(),
-      this.buttonMode = ButtonMode.MATERIAL});
+      this.buttonMode = ButtonMode.MATERIAL,
+      this.padding = EdgeInsets.zero});
```

Comment:
Can we rename the property to `gridPadding` to give more context and place it below `horizontalSpacing` ?

Reply:
I have modified my code, please check it. 😄